perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -86,6 +94,19 @@
                      'Allow swiftclient to access Swift service without \n'
                      'having to verify the SSL certificate')
 
+flags.DEFINE_integer('object_storage_multistream_objects_per_stream', 1000,
+                     'Number of objects to send and/or receive per stream. '
+                     'Only applies to the api_multistream scenario.',
+                     lower_bound=1)
+flag_util.DEFINE_yaml('object_storage_object_sizes', '1KB',
+                      'Size of objects to send and/or receive. Only applies to '
+                      'the api_multistream scenario. Example: {1KB: 100%}')
```

Comment:
The example here is for how to specify a distribution. Please clarify that you can also specify a single size and give an example here